janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/formats/cassandra/CqlBridgeRecordReader.java

```diff
@@ -0,0 +1,377 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
I think we just need to uphold the terms of the original ASF Apache license, which in this case means we preserve the license in the file as well as any other relevant copyright information in their NOTICE file. But in this case I don't see anything relevant in the notice file with the possible exception of formally stating google-collection's copyright information. I don't think we can infer copyright ownership from the git commit log. For example some authors may have committed as part of an organization and may not be individually entitled to claim copyright for their work.